src/DataGrid.tsx

```diff
@@ -368,43 +550,130 @@ function DataGrid<R, K extends keyof R, SR>({
     }
   }
 
-  function getViewportRows() {
-    const rowElements = [];
+  function navigate(key: string, shiftKey: boolean) {
+    let nextPosition = getNextPosition(key, shiftKey, selectedPosition);
+    let mode = cellNavigationMode;
+    if (key === 'Tab') {
```

Comment:
When `Tab` is being held, you will see the following that the tab focus element is not moving to the next possible cell but only focus on the select `input`. Not a big deal, but we might need to revisit later.

![tab gap](https://user-images.githubusercontent.com/25043398/86829502-19c2ae00-c05a-11ea-9fae-b0d9c3117078.gif)

Reply:
Fixed